packages/uikit-chat-hooks/src/channel/useGroupChannelMessages/useGroupChannelMessagesWithQuery.ts

```diff
@@ -60,7 +61,7 @@ export const useGroupChannelMessagesWithQuery: UseGroupChannelMessages = (sdk, c
     async (uid?: string) => {
       if (uid) {
         queryRef.current = createMessageQuery(channel, options?.queryCreator);
-        channelMarkAsRead().catch();
+        channelMarkAsRead().catch(NOOP);
```

Comment:
Is there any reason to keep calling catch?

Reply:
Not important though, it is to avoid unhandled promise warning logs. (In React Native, warning and error logs show as pop-ups on the device screen)

But..🤔 it seems a catch handler is not needed when this case! (already catch used in channelMarkAsRead function) Thanks!